src/ts/components/AppBar.tsx

```diff
@@ -0,0 +1,76 @@
+import React, {Component} from 'react';
```

Comment:
let's keep the spaces on both sides of the curly braces, everywhere except in jsx/tsx expressions, like so:
```
import React, { Component } from 'react'
```